Code/Tools/AssetProcessor/native/AssetManager/SourceFileRelocator.cpp

```diff
@@ -387,12 +397,15 @@ Please note that only those seed files will get updated that are active for your
                     return AZ::Failure(AZStd::string::format("Path %s points to a folder outside the current project's scan folders.\n", pathOnly.c_str()));
                 }
 
-                GetFilesFromSourceControl(sources, scanFolderInfoOut, normalizedSource.c_str(), excludeMetaDataFiles);
+                fileExists = GetFilesFromSourceControl(sources, scanFolderInfoOut, normalizedSource.c_str(), excludeMetaDataFiles);
             }
 
             if(sources.empty())
             {
-                return AZ::Failure(AZStd::string("Wildcard search did not match any files.\n"));
+                if (fileExists)
+                    return AZ::Failure(AZStd::string("Wildcard search did not match any files found in the asset database.\n"));
```

Comment:
is there any way to include the file pattern or any input that might have caused this to fail?

Reply:
Also the code standard is to include curly braces for each logical expression:
if (foo)
{
// do logic
}
https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#use-of-curly-braces